### DISCUSSION TITLE
A possibility to update a beatmap without deleting the custom difficulties in it

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#24414] [Ideas]
This wasn't an issue in Stable as there was per-diff update button
In lazer if you update the mapset that has a custom non-uploaded diff in it, the diff gets deleted.

**Comment by Loreos7** [comment]
Yep. I agree. Copied diffs shouldnt be deleted while update.

https://github.com/ppy/osu/assets/86934170/71fe0b18-0f3b-4d0c-8149-2f804bc6b072

P.S. Pop-up dialog has been shown because I also modified "Insane" diff.
